reference/orders.v2.oas2.yml

```diff
@@ -283,6 +306,38 @@ paths:
             schema:
               $ref: '#/components/schemas/order_Post'
             examples:
+              Order with Consignments:
+                value:
+                  status_id: 0
+                  customer_id: 12
+                  billing_address: []
+                  cart: []
+                  channelID: 1
+                  consignments:
+                    pickups:
+                      - id: 99
+                        pickup_method_id: 1
+                        pickup_method_display_name: Pick Up
+                        collection_instructions: Bring your ID
+                        collection_time_description: 9am - 6pm
+                        location:
+                          id: 1
+                          name: Location 1
+                          code: LOCATION-1
+                          address_line_1: 123 Main Street
+                          address_line_2: Suite 101
+                          city: Austin
+                          state: Texas
+                          postal_code: '78726'
+                          country_alpha2: US
+                          email: location1@example.com
+                          phone: +1 111-111-1111
+                        line_items:
+                          - url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/129/products/12'
+                            resource: /orders/129/products/12
```

Comment:
This isn't valid for a request body. This array needs to contain actual line item data